Compilation error in AbstractWebAuthnAccountTest

### Before reporting an issue

- [x] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

testsuite

### Describe the bug

We have commited two PR that adds the same page:

```
[ERROR] /home/rmartinc/keycloak/testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/webauthn/account/AbstractWebAuthnAccountTest.java:[80,33] variable webAuthnErrorPage is already defined in class org.keycloak.testsuite.webauthn.account.AbstractWebAuthnAccountTest
```

### Version

999.0.0-SNAPSHOT

### Regression

- [ ] The issue is a regression

### Expected behavior

Should compile

### Actual behavior

Compilation error

### How to Reproduce?

mvn clean install -DskipTests

### Anything else?

_No response_

--- a/testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/webauthn/account/AbstractWebAuthnAccountTest.java
+++ b/testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/webauthn/account/AbstractWebAuthnAccountTest.java
@@ -34,7 +34,6 @@ import org.keycloak.representations.idm.RequiredActionProviderSimpleRepresentati
 import org.keycloak.testsuite.AbstractAuthTest;
 import org.keycloak.testsuite.page.AbstractPatternFlyAlert;
 import org.keycloak.testsuite.webauthn.pages.SigningInPage;
-import org.keycloak.testsuite.webauthn.pages.WebAuthnErrorPage;
 import org.keycloak.testsuite.webauthn.utils.SigningInPageUtils;
 import org.keycloak.testsuite.pages.DeleteCredentialPage;
 import org.keycloak.testsuite.updaters.RealmAttributeUpdater;
@@ -75,9 +74,6 @@ public abstract class AbstractWebAuthnAccountTest extends AbstractAuthTest imple
 
     @Page
     private DeleteCredentialPage deleteCredentialPage;
-
-    @Page
-    protected WebAuthnErrorPage webAuthnErrorPage;
 
     private VirtualAuthenticatorManager webAuthnManager;
     protected SigningInPage.CredentialType webAuthnCredentialType;

